tests/test_graphical_units/img_svg_resources/logo.svg

```diff
@@ -0,0 +1,44 @@
+<svg version="1.1" viewBox="0.0 0.0 800.0 800.0" fill="none" stroke="none" stroke-linecap="square"
+     stroke-miterlimit="10" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns="http://www.w3.org/2000/svg">
+    <clipPath id="p.0">
```

Comment:
Is this one free of copyrights?

Reply:
same as the discussion above - I think it'll be best to get approval from anyone who posted them.